Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -746,6 +747,13 @@ void LoadILLSANS::loadMetaData(const NeXus::NXEntry &entry,
   // Add a log called timer with the value of duration
   const double duration = entry.getFloat("duration");
   runDetails.addProperty<double>("timer", duration);
+
+  // Get the starting date of the measure, as it is needed by the instrument and
+  // parameter loaders later
+  std::string startDate = entry.getString("start_time");
+  LoadHelper loadHelper = LoadHelper();
```

Comment:
Just an early comment, this should have a member variable of a loadHelper, can you use that instead?